src/types.rs

```diff
@@ -144,54 +144,87 @@ impl From<Operator> for Atom {
 
 /// Records a universe of symbols.
 ///
-/// Use [`Signature::default`] for a blank signature, or [`Signature::new`] to initialize a
-/// signature with given operators.
+/// Use [`Signature::default`] for a blank `Signature`, or [`Signature::new`] to initialize a
+/// `Signature` with given [`Operator`]s.
 ///
 /// [`Signature::default`]: #method.default
 /// [`Signature::new`]: #method.new
+/// [`Operator`]: struct.Operator.html
+///
+/// # Examples
+///
+/// ```
+/// # use term_rewriting::{Signature, parse_term};
+/// // Constructing a Signature using the default
+/// let mut sig1 = Signature::default();
+/// let a = sig1.new_op(2, Some("A".to_string()));
+/// let b = sig1.new_op(0, Some("B".to_string()));
+/// let c = sig1.new_op(0, Some("C".to_string()));
+///
+/// // Constructing a Signature using Signature::new
+/// let (mut sig2, _) = Signature::new(vec![
+///     (2, Some("A".to_string())),
+///     (0, Some("B".to_string())),
+///     (0, Some("C".to_string())),
+/// ]);
+///
+/// // Constructing a Signature using the new_* functions.
+/// let mut sig3 = Signature::default();
+/// let a = sig3.new_op(2, Some("A".to_string()));
+/// let b = sig3.new_op(0, Some("B".to_string()));
+/// let c = sig3.new_op(0, Some("C".to_string()));
+///
+/// assert_eq!(sig1.clone(), sig2);
+/// assert_eq!(sig1, sig3);
```

Comment:
What's the difference between the first and third methods here? If none, could you remove one?